subprojects/packagefiles/lcms2/meson.build

```diff
@@ -0,0 +1,65 @@
+project('Little-CMS', 'c',
+  version: '2.12'
+)
+
+cc = meson.get_compiler('c')
+
+lcms2_srcs = [ 'src/cmsalpha.c'
+        , 'src/cmscgats.c'
+        , 'src/cmserr.c'
+        , 'src/cmsgmt.c'
+        , 'src/cmsintrp.c'
+        , 'src/cmsio1.c'
+        , 'src/cmsmd5.c'
+        , 'src/cmsnamed.c'
+        , 'src/cmspack.c'
+        , 'src/cmsplugin.c'
+        , 'src/cmssamp.c'
+        , 'src/cmstypes.c'
+        , 'src/cmswtpnt.c'
+        , 'src/cmscam02.c'
+        , 'src/cmscnvrt.c'
+        , 'src/cmsgamma.c'
+        , 'src/cmshalf.c'
+        , 'src/cmsio0.c'
+        , 'src/cmslut.c'
+        , 'src/cmsmtrx.c'
+        , 'src/cmsopt.c'
+        , 'src/cmspcs.c'
+        , 'src/cmsps2.c'
+        , 'src/cmssm.c'
+        , 'src/cmsvirt.c'
+        , 'src/cmsxform.c'
+]
+
+inc_dirs = include_directories('include')
+
+deps = [
+  cc.find_library('m', required: false),
+  dependency('threads'),
+  dependency('zlib'),
+  dependency('libtiff-4'),
+]
+
+cargs = ['-DHasTHREADS=1', '-DHasZLIB=1', '-DHasTIFF=1']
+
+if cc.has_argument('-fvisibility')
```

Comment:
Would it make more sense to use `cc.has_function_attribute('visibility')` here?

Reply:
Pretty much the same thing, I copy that from another project...